src/styles/styles.css

```diff
@@ -0,0 +1,306 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;600;700;800;900&display=swap');
+
+body {
+  width: 100%;
+  height: auto;
+  background-color: #f5f6fa;
+  text-decoration: none;
+}
```

Comment:
![image](https://github.com/demesameneshoa/book-store/assets/60083437/e59bb8a8-320f-4dd3-a9df-381cba8fe4a8)

- [ ] You can see the contents are not perfectly aligned to the `left side` and the `right side`. Kindly fix this issue to match the given design guideline.